pkg/components/helpers.go

```diff
@@ -138,8 +137,16 @@ func CreateUser(ctx context.Context, ytClient yt.Client, userName, token string,
 }
 
 func IsUpdatingComponent(ytsaurus *apiproxy.Ytsaurus, component Component) bool {
-	componentNames := ytsaurus.GetLocalUpdatingComponents()
-	return (componentNames == nil && component.IsUpdatable()) || slices.Contains(componentNames, component.GetName())
+	components := ytsaurus.GetUpdatingComponents()
+	if components == nil && component.IsUpdatable() {
```

Comment:
let's check len(components) just in case, I think it is a best practice not to distinguish nil and empty slices.